.travis.yml

```diff
@@ -5,8 +5,9 @@ python:
     - 2.7
     - 3.3
     - 3.4
+    - pypy
```

Comment:
There was a syntax error with the Unicode string prefixes. I guess I can
try to add all the compatibility shims.

Sent from my phone

On Jun 1, 2015, at 12:21, Tres Seaver <notifications@github.com> wrote:

In .travis.yml
<https://github.com/zopefoundation/zope.session/pull/1#discussion_r31447792>
:

> @@ -5,8 +5,9 @@ python:
>      - 2.7
>      - 3.3
>      - 3.4
> +    - pypy

We probably want pypy3 too (or document why not).

—
Reply to this email directly or view it on GitHub
<https://github.com/zopefoundation/zope.session/pull/1/files#r31447792>.